src/modules/dailyCreditsSummary/i18n/pl_PL.ts

```diff
@@ -0,0 +1,106 @@
+        },
+        mission: {
+            regex: '^%misje%( \\(System alarmu przeciwpożarowego\\))?( - Fałszywy alarm)?$',
+            title: 'Własne misje',
+            backgroundColor: '#ff2400',
+            textColor: 'Biały',
+        },
+        cancelledMissions: {
+            regex: '^%misje%( \\(System alarmu przeciwpożarowego\\))? - [aA]nulowane$',
+            title: 'Anulowane misje',
+            backgroundColor: '#7C7978',
+            textColor: 'Biały',
+        },
+        buyVehicle: {
+            regex: /Kupiony pojazd/u,
+            backgroundColor: '#007fff',
+            textColor: 'Biały',
+        },
+        dailyLogin: {
+            regex: /Nagroda za codzienne logowanie/u,
+            backgroundColor: '#8db600',
+            textColor: 'Czarny',
+        },
+        buildings: {
+            regex: /( Zbudowane budynki|Budynki usunięte bez zwrotu|Zwrot kosztów budynku$)/u,
+            title: 'Zbudowano budynek/usunięte',
+            backgroundColor: '#ed872d',
+            textColor: 'Czarny',
+        },
+        upgradeBuilding: {
+            regex: /(Stacja (ulepszona( \(z małej .*? stacji\))??|zbudowana)|Rozszerzona ochrona|Anuluj:|Zwrot kosztów aktualizacji budynku)/u,
+            title: 'Zbudowana stacja',
+            backgroundColor: '#000080',
+            textColor: 'Biały',
+        },
+        task: {
+            regex: /Ukończone zadanie («.*?»|".*?")/u,
+            title: 'Ukończone zadanie',
+            backgroundColor: '#bb3385',
+            textColor: 'Biały',
+        },
+        patients: {
+            regex: /Pacjent (Transport|Opieka)/u,
+            title: 'Pacjenci',
+            backgroundColor: '#fff600',
+            textColor: 'Czarny',
+        },
+        alliancePatients: {
+            regex: /Szpital - Sojuszu/u,
+            backgroundColor: '#ffff99',
+            textColor: 'Czarny',
+        },
+        alliancePrisoners: {
+            regex: /Policja - Sojuszu/u,
+            backgroundColor: '#ffff99',
+            textColor: 'Czarny',
+        },
+        prisoners: {
+            regex: /Więzień? Przewieziony/u,
+            title: 'Więzień',
+            backgroundColor: '#0bda51',
+            textColor: 'Czarny',
+        },
+        schoolings: {
+            regex: /(Kandydat na szkolenie Sojuszu|Edukacja)/u,
+            title: 'Edukacja',
+            backgroundColor: '#ff00ff',
+            textColor: 'Biały',
+        },
+        shitComplexes: {
+            regex: /(Budynek dołączony do Bazy Kompleksu Budynków|Ulepszony do bazy kompleksu budynków|Ulepszony do kompleksu budynków|Ukończono rozbudowę bazy kompleksu budynków)/u,
+            title: 'Kompleks budynków',
+            backgroundColor: '#b5651d',
+            textColor: 'Biały',
+        },
+        fireAlarmSystemCancel: {
+            regex: /Fałszywy alarm/u,
+            titel: 'Fałszywy alarm',
+            backgroundColor: '#c80815',
+            textColor: 'Biały',
+        },
```

Comment:
that is a bit more complex and need to match the rows in https://www.operatorratunkowy.pl/credits/daily
We will do that after the first merge